pkg/security/probe/process_resolver.go

```diff
@@ -598,6 +598,11 @@ func (p *ProcessResolver) resolveWithKernelMaps(pid, tid uint32) *model.ProcessC
 	return p.insertExecEntry(pid, entry)
 }
 
+// IsKThread returns whether given pids are from kthreads
+func IsKThread(ppid, pid uint32) bool {
+	return ppid == 2 || pid == 2
```

Comment:
Is this a sure way to check for kthreads ?

Reply:
yes I double checked